.github/workflows/build.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
         with:
-          python-version: "3.10"
+          python-version-file: main/.python-version
```

Comment:
Why is this moved to a file? Is it because of mise?

Reply:
It's to have a single definition for the python-version. Works with mise as well